charts/lagoon-core/Chart.yaml

```diff
@@ -54,3 +54,5 @@ annotations:
       description: ssh, ssh-token and broker service configuration options
     - kind: changed
       description: update nats to v1.2.11
+    - kind: changed
+      description: Lagoon Workflows subsystem removed
```

Comment:
Since we changed to the model of merging to main and then doing a release of the charts as a separate version bump. You'll need to remove all the previous changelog entries from this pullrequest, since all the other changes were part of chart `v1.51.0`. The same applies to the `lagoon-test` chart yaml.